spec/quality/tools/bigfiles_spec.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+require_relative '../../spec_helper'
+require 'quality/tools/bigfiles'
+
+describe Quality::Tools::Bigfiles do; end
```

Comment:
[rubocop] Style/BlockDelimiters: Prefer `{...}` over `do...end` for single-line blocks.